### PR TITLE
Update S3 directions

### PR DESCRIPTION
**Pull request by Greenwoodmatthew**

### CODE REVIEW

**Comment by Greenwoodmatthew** [comment]
Review to merge this request which
- Looks to sperate incoming and outgoing requests into different guides
- Removes internal guidelines from the public repo and makes a new internal guidance repo
Still needs updates, but i feel if we merge this in the set the structure we can then focus on each elements as the requests come in?

File: s3-to-s3-outgoing.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                "s3:GetObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::SOURCE_BUCKET_NAME/*",
+                "arn:aws:s3:::SOURCE_BUCKET_NAME"
+            ]
+        }
+    ]
+}
+```
```
Comment (KellyVance) [inline s3-to-s3-outgoing.md:51]: The original may be the correct AWS jargon, so feel free to ignore the below. But a GSQ source account that sends data to a source bucket is a confusing thing to read. 

```suggestion
Attaching this policy to the *destination* S3 bucket allows the GSQ *source* account to perform the ListBucket and GetObject commands on the *destination* S3 bucket, and to copy the objects across.  

1. Sign in to your *destination* AWS account.  
2. Attach the S3 bucket policy sent to you by GSQ to the *destination* bucket (see [how-to](https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html)).  
3. An example of the bucket policy is as per the below code. However, the policy we send to you will have the value of **Principal** set to GSQ's ARN and will be customised to have your source bucket name listed.

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "DelegateS3Access",
            "Effect": "Allow",
            "Principal": {
                "AWS": "arn:aws:iam::GSQ_ARN_ACCOUNT_NUMBER:root"
            },
            "Action": [
                "s3:ListBucket",
                "s3:GetObject"
            ],
            "Resource": [
                "arn:aws:s3:::DESTINATION_BUCKET_NAME/*",
                "arn:aws:s3:::DESTINATION_BUCKET_NAME"
            ]
        }
    ]
}
```
```

File: s3-to-s3-outgoing.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# How to export data from GSQ using S3 to S3 sync
```
Comment (KellyVance) [inline s3-to-s3-outgoing.md]: If this is for outward facing client perspective
```suggestion
# How to import data from GSQ using S3 to S3 sync
```

**Comment by KellyVance** [comment]
@LukeHauck FYI

Will eventually tie into the Transfer Family work

**Review by katewd — APPROVED** [review]
Splitting instructions by incoming and outgoing works well.

**Pull request merged**